### PR TITLE
Added a tutorial page for VPA in kkp

### PR DESCRIPTION
**Pull request by dharapvj**
Currently, VPA feature in KKP is not well documented. So this PR adds some initial documentation on how to make use of this feature.

### CODE REVIEW

**Comment by dharapvj** [comment]
CC: @toschneck - FYI

File: content/kubermatic/main/tutorials-howtos/kkp-autoscaler/vertical-pod-autoscaler/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+date = 2024-09-25T14:07:10+02:00
+weight = 9
++++
+
+This section explains how Kubernetes Vertical Pod Autoscaler helps in a scaling the control plane components for user-clusters with rising load on that user-cluster.
+
+## What is a Vertical Pod Autoscaler in Kubernetes?
+[Vertical Pod Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) (VPA) frees users from the necessity of setting up-to-date resource limits and requests for the containers in their pods. When configured, it will set the requests automatically based on usage and thus allow proper scheduling onto nodes so that appropriate resource amount is available for each pod. It will also maintain ratios between limits and requests that were specified in initial containers configuration.
+
+Unlike HPA, VPA does not bundled with kubernetes itself. It must get installed in the cluster separately.
```
Comment (xrstf) [inline content/kubermatic/main/tutorials-howtos/kkp-autoscaler/vertical-pod-autoscaler/_index.en.md:12]: uppercase Kubernetes

File: content/kubermatic/main/tutorials-howtos/kkp-autoscaler/vertical-pod-autoscaler/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+1. prometheus
+1. scheduler
+
+All these components have default resources allocated by KKP. You can either use [`componentsOverride` section](../../../tutorials-howtos/operation/control-plane/scaling-the-control-plane/#setting-custom-overrides) of cluster resource to override the resources manually, or let VPA handle it for you!
+
+> Note: If you enable VPA and add `componentsOverride` block as well for a given cluster to specify resources, `componentsOverride` takes precedence.
+
+## How to enable VPA in KKP
+To enable VPA controlled control-plane components for user-clusters, we just need to turn on a featureFlag in Kubermatic Configuration.
+```
```
Comment (xrstf) [inline content/kubermatic/main/tutorials-howtos/kkp-autoscaler/vertical-pod-autoscaler/_index.en.md:33]: This can be tagged as YAML

Comment (xrstf) [reply content/kubermatic/main/tutorials-howtos/kkp-autoscaler/vertical-pod-autoscaler/_index.en.md:33]: And please leave an empty line before and after a code block.

**Review by xrstf — CHANGES_REQUESTED** [review]
Some small nitpickery

**Comment by dharapvj** [comment]
I have fixed those nits. please recheck

**Comment by ahmedwaleedmalik** [comment]
@dharapvj KKP 2.26 docs were branched out before this PR was merged. We need to copy these docs to [v2.26 directory](https://github.com/kubermatic/docs/tree/main/content/kubermatic/v2.26) as well.

**Pull request merged**